test/test_heal.sh

```diff
@@ -8,7 +8,9 @@ exec 3>"$input"
 
 cat >> "$input" <<EOF
 players[1].ship.corePart.modules.hull.health = 1
-players[1].ship.corePart.modules.heal.timer.time = 0.1
+timer = players[1].ship.corePart.modules.heal.timer
+timer.time = 0.1
+players[1].ship.corePart.modules.heal.timer = timer
```

Comment:
It's not possible to just modify values inside a nested object and have them affect the real object in Rust. When we fetch the timer object, Rust copies it. When we set the timer object on the heal module, it copies it back. It might be possible to do something fancy with a bit more work on the Rust side, but this will only be used for tests, so it should be fine.